spm_results_export.m

```diff
@@ -231,7 +231,9 @@
 fclose(fid);
 %==========================================================================
 %-Delete temporary files
-
+if exist('OCTAVE_VERSION','builtin')
+    confirm_recursive_rmdir(false,'local');
+end

```

Comment:
Hi @gllmflndn , I was just wondering the purpose of this if statement? Would I be right in thinking it prevents the code asking the user's permission to remove directories?

Reply:
Ah okay I see! Thank you for explaining.